### PR TITLE
DOMA-1587 fixed tin validation

### PR DESCRIPTION
**Pull request by akarjakin**
* spaces are not a problem anymore
* refactoring
* fixed/added tests
* updated string utils

image

### CODE REVIEW

File: apps/condo/domains/organization/utils/tin.utils.spec.js [hunk 1]
```diff
@@ -2,15 +2,23 @@ const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
 const { isValidTin } = require('@condo/domains/organization/utils/tin.utils')
 
 const VALID_RU_TIN_10 = '1654019570'
+const VALID_RU_TIN_10_1 = '6311095616'
+const VALID_RU_TIN_10_SPC = '    1654019570    '
```
Comment (SavelevMatthew) [inline apps/condo/domains/organization/utils/tin.utils.spec.js:6]: Please add more cases…like tabulations, non-breaking spaces and so

**Review by SavelevMatthew — APPROVED** [review]
LGTM…but other cases with space-like symbols can be helpful too

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/domains/organization/utils/tin.utils.spec.js (hunk 1 above)
Comment (akarjakin) [reply apps/condo/domains/organization/utils/tin.utils.spec.js:6]: done

File: apps/condo/domains/organization/utils/tin.utils.spec.js [hunk 2]
```diff
@@ -2,15 +2,23 @@ const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
...
 const VALID_RU_TIN_12 = '500110474504'
 const INVALID_RU_TIN_10 = '01234556789'
 const INVALID_RU_TIN_12 = '0123455678901'
 const SOME_RANDOM_LETTERS = 'ABCDEFGHIJ'
 
 describe('isValidTin()', () => {
-    test('for valid 10 char RU INN ', () => {
+    test('for valid 10 char RU INN #1', () => {
```
[the remark this answers is not in the document]
Comment (akarjakin) [reply apps/condo/domains/organization/utils/tin.utils.spec.js:13]: done

**Pull request merged**